mmdet/models/detectors/kd_one_stage.py

```diff
@@ -30,11 +33,13 @@ def __init__(self,
                          pretrained)
 
         self.eval_teacher = eval_teacher
-
-        from mmdet.apis.inference import init_detector
-
-        self.teacher_model = init_detector(
-            teacher_config, teacher_ckpt, device='cpu')
+        # Build teacher model
+        if isinstance(teacher_config, str):
+            teacher_config = mmcv.Config.fromfile(teacher_config)
+        self.teacher_model = build_detector(teacher_config.model)
```

Comment:
Better use `teacher_config['model']`, otherwise it is incompatible with dict input type for `teacher_config`.